tests/language-service/languageservicerunner.ts

```diff
@@ -123,13 +140,15 @@ function getTestCases() {
     return testCases;
 }
 
+const fileName = (isPython: boolean) => isPython ? "main.py" : "main.ts"
```

Comment:
This is not new but what about other files, does it matter?

Reply:
This is just for shimming the compile options. We assume each test case can be in a single file, which I don't see any need to support more. We have a "test package" already that lets us add arbitrary amounts of support files.